data/factory-sets.json

```diff
@@ -7149,215 +7017,689 @@
+				"nature": "Adamant",
+				"moves": [["Stone Edge"], ["Earthquake"], ["Megahorn"], ["Ice Punch"]]
+			}, {
+				"species": "Rhyperior",
+				"gender": "",
+				"item": ["Leftovers", "Rindo Berry"],
+				"ability": ["Solid Rock"],
+				"evs": {"hp": 248,"atk": 16,"def": 0,"spa": 0,"spd": 244,"spe": 0},
+				"nature": "Adamant",
+				"moves": [["Earthquake"], ["Rock Blast"], ["Ice Punch"], ["Stealth Rock"]]
+			}]
+		},
+		"ribombee": {
+			"flags": {},
+			"sets": [{
+				"species": "Ribombee",
+				"gender": "",
+				"item": ["Life Orb"],
+				"ability": ["Shield Dust"],
+				"evs": {"hp": 0,"atk": 0,"def": 0,"spa": 252,"spd": 4,"spe": 252},
+				"ivs": {"hp": 31,"atk": 0,"def": 31,"spa": 31,"spd": 31,"spe": 31},
 				"nature": "Timid",
-				"moves": [["Nasty Plot"], ["Shadow Ball"], ["Focus Blast"], ["Substitute"]]
+				"moves": [["Quiver Dance"], ["Moonblast"], ["Sticky Web"], ["Psychic"]]
+			}]
+		},
+		"roserade": {
+			"flags": {},
+			"sets": [{
+				"species": "Roserade",
+				"gender": "",
+				"item": ["Life Orb", "Grassium Z"],
+				"ability": ["Technician"],
+				"evs": {"hp": 0,"atk": 0,"def": 4,"spa": 252,"spd": 0,"spe": 252},
+				"nature": "Timid",
+				"moves": [["Spikes"], ["Hidden Power Fire"], ["Sludge Bomb", "Sleep Powder"], ["Leaf Storm"]]
+			}, {
+				"species": "Roserade",
+				"gender": "",
+				"item": ["Choice Scarf"],
+				"ability": ["Technician"],
+				"evs": {"hp": 0,"atk": 0,"def": 4,"spa": 252,"spd": 0,"spe": 252},
+				"nature": "Timid",
+				"moves": [["Hidden Power Rock"], ["Extrasensory", "Sludge Bomb"], ["Leaf Storm"], ["Sleep Powder"]]
+			}, {
+				"species": "Roserade",
+				"gender": "",
+				"item": ["Choice Scarf"],
+				"ability": ["Technician"],
+				"evs": {"hp": 0,"atk": 0,"def": 4,"spa": 252,"spd": 0,"spe": 252},
+				"nature": "Timid",
+				"moves": [["Hidden Power Rock"], ["Extrasensory", "Sludge Bomb"], ["Leaf Storm"], ["Spikes"]]
+			}]
+		},
+		"rotommow": {
+			"flags": {},
+			"sets": [{
+				"species": "Rotom-Mow",
+				"gender": "",
+				"item": ["Grassium Z", "Iapapa Berry"],
+				"ability": ["Levitate"],
+				"evs": {"hp": 0,"atk": 0,"def": 4,"spa": 252,"spd": 0,"spe": 252},
+				"nature": "Timid",
+				"moves": [["Leaf Storm"], ["Volt Switch"], ["Defog"], ["Will-O-Wisp", "Pain Split"]]
+			}]
+		},
+		"rotomheat": {
+			"flags": {},
+			"sets": [{
+				"species": "Rotom-Heat",
+				"gender": "",
+				"item": ["Iapapa Berry", "Firium Z"],
+				"ability": ["Levitate"],
+				"evs": {"hp": 4,"atk": 0,"def": 0,"spa": 252,"spd": 0,"spe": 252},
+				"nature": "Timid",
+				"moves": [["Overheat"], ["Will-O-Wisp"], ["Defog"], ["Volt Switch"]]
+			}, {
+				"species": "Rotom-Heat",
+				"gender": "",
+				"item": ["Iapapa Berry", "Firium Z"],
+				"ability": ["Levitate"],
+				"evs": {"hp": 4,"atk": 0,"def": 0,"spa": 252,"spd": 0,"spe": 252},
+				"nature": "Timid",
+				"moves": [["Overheat"], ["Hidden Power Ice"], ["Defog"], ["Volt Switch"]]
```

Comment:
Please stop doing stuff like this, it's an incredible waste of space. The above set is literally identical except this one has Hidden Power Ice over Will-O-Wisp; just do `["Will-O-Wisp", "Hidden Power Ice"]`.

I understand not doing it if it changes the frequency of a set appearing, since there's no system in place for that, but when every set is identical it makes no sense to do it this way.